api/openapi.yaml

```diff
@@ -873,6 +873,521 @@ paths:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StudyResult"
+    patch:
+      tags:
+        - studies
+      summary: update an operational study
+      parameters:
+        - in: query
+          name: project_id
+          schema:
+            type: integer
+          description: project id refered to the study
+        - in: query
+          name: study_id
+          schema:
+            type: integer
+          description: study id you want to retrieve
+      requestBody:
+        description: The fields you want to update
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/StudyRequest"
+      responses:
+        200:
+          description: The operational study updated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StudyResult"
+    delete:
+      tags:
+        - studies
+      summary: delete an operational study
+      parameters:
+        - in: query
+          name: project_id
+          schema:
+            type: integer
+          description: project id refered to the operational study
+        - in: query
+          name: study_id
+          schema:
+            type: integer
+          description: study id you want to delete
+      responses:
+        204:
+          description: No content
+  /projects/{project_id}/study_types/:
+    get:
+      tags:
+        - studies
+      summary: Retrieve list of study types
+      parameters:
+        - in: query
+          name: project_id
+          schema:
+            type: integer
+          description: project id refered to the operational study
+      responses:
+        200:
+          description: The list of study types
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                example:
+                  [
+                    "timeTables",
+                    "flowRate",
+                    "parkSizing",
+                    "garageRequirement",
+                    "operationOrSizing",
+                    "operability",
+                    "strategicPlanning",
+                    "chartStability",
+                    "disturbanceTests",
+                  ]
+  /projects/{project_id}/study_states/:
+    get:
+      tags:
+        - studies
+      summary: Retrieve list of study states
+      parameters:
+        - in: query
+          name: project_id
+          schema:
+            type: integer
+          description: project id refered to the operational study
+      responses:
+        200:
+          description: The list of study states
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+                example: ["started", "inProgress", "finish"]
```

Comment:
Study states should be part of the study model. So an endpoint like that is not necessary.
We can keep it that way and create an issue to implement dynamic states.